lib/demon/email_sync.rb

```diff
@@ -3,9 +3,12 @@
 require "demon/base"
 
 class Demon::EmailSync < ::Demon::Base
-  HEARTBEAT_KEY = "email_sync_heartbeat"
   HEARTBEAT_INTERVAL = 60.seconds
 
+  def self.heartbeat_key
+    "#{HOSTNAME}_#{Process.pid}_email_sync_heartbeat"
```

Comment:
I think the unique per site key was intentional to ensure a single instance of EmailSync is running.